### DISCUSSION TITLE
Tuya items not showing up in Homebridge

### DISCUSSION THREAD

**Discussion by Wdave40** [tuya/tuya-homebridge#189] [Q&A]
Hi, I am really new at this so please bear with me. 
I have Homebridge successfully running on a pi4, I set up the Tuya dev account and installed the app on my iPhone. When I add items on my phone in the Tuya app it shows up in the dev platform. I followed all the Tuya install instructions on 2 youtube videos, the plugins are there but none of my devices on tuya are showing up in Homebridge. I set up the plugin and the bridge too. The ports seem to be correct. After several hours I am at a loss. 

Thanks for any help!

**Comment by Wdave40** [comment]
Once I clicked on debug then it showed the password I had in the app and the one in the plugin were different. Once I corrected that it all worked!

**Reply by VES-ES** [reply]
Newbie here too. Where did you click on debug?
